fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -266,6 +272,50 @@ public boolean isAvailable() {
         return this.isAlive.get() && !this.isDecommissioned.get();
     }
 
+    public int getNumRunningQueries() {
+        return numRunningQueries;
+    }
+
+    public long getMemUsedBytes() {
+        return memUsedBytes;
+    }
+    public long getMemLimitBytes() {
+        return memLimitBytes;
+    }
+
+    public double getMemUsedPct() {
+        if (0 == memLimitBytes) {
+            return 0;
+        }
+        return ((double) memUsedBytes) / memLimitBytes;
+    }
+
+    public int getCpuUsedPermille() {
+        return cpuUsedPermille;
+    }
+
+    public boolean updateResourceUsage(int numRunningQueries, long memLimitBytes, long memUsedBytes,
```

Comment:
isChanged should always be true, just update is ok.

Reply:
If there isn't any query running, then `isChanged` will be false.